src/components/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { Button } from '@/components/_buttons/Button'
+import { usePersonalDataStore } from '@/providers/personalData'
+
+import { Banner, BannerProps } from '.'
+import { SvgActionWarning, SvgAlertsInformative24 } from '../_icons'
+
+export default {
+  title: 'other/Banner',
+  component: Banner,
+  args: {
+    id: 'some-id',
+    description:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quod, animi tempore dicta vero temporibus debitis.',
+    title: 'Banner Title',
+    icon: <SvgAlertsInformative24 />,
+  },
+} as Meta
+
+const Template: Story<BannerProps> = (args) => {
+  const updateDismissedMessages = usePersonalDataStore((state) => state.actions.updateDismissedMessages)
+  return (
+    <>
+      <Banner {...args} />
+      <Button
+        onClick={() => {
+          updateDismissedMessages(args.id, false)
+        }}
+      >
+        Reset
+      </Button>
```

Comment:
Maybe add some margin here so the button is not completely attached to the banner